frontend/src/worksheets/TableListOfWorkSheets.js

```diff
@@ -25,6 +33,26 @@ export default function TableListOfWorkSheets() {
           <Button text="Új munkalap létrehozása" moreClassName="w-auto p-1" />
         </Link>
       </div>
+      <div>Szűrés dátum szerint:</div>
+      <div>
+        <CalendarDropDown
+          date={startDate}
```

Comment:
Ez tud date range-et is (https://reactdatepicker.com/#example-date-range-for-one-datepicker), lehet felhasználóbarátabb lenne.
Ez meg még szebb és pro-bb: https://hypeserver.github.io/react-date-range/
Egyeztessetek PM-mel UX-et illetően. Btw, az issue-hoz ő is a dateranget írta kívánatosnak.

Reply:
Én úgy láttam, hogy a range úgy működik, hogy kijelölöm a kívánt napokat. De az csak akkor praktikus, ha pár napot akarok kijelölni, és nem pl. 2 évet. Tamás szerintem arra értette a range-et, hogy ne 1 dátumot tudjunk megadni, hanem intervallumot (de lehet, hogy félreértettem).

Reply:
Akkor jó lesz így egyelőre.